pkg/framework/env_var.go

```diff
@@ -107,6 +107,17 @@ func SetEnvVar(key, value string, container *corev1.Container) {
 	container.Env = append(container.Env, corev1.EnvVar{Name: key, Value: value})
 }
 
+// AppendEnvVar will update or add the environment variable
+func AppendEnvVar(key, value string, envs []corev1.EnvVar) []corev1.EnvVar {
```

Comment:
Seems to duplicate EnvOverride function above.

Reply:
Although the EnvOverride is using EnvVar as parameter.
You can construct EnvVar parameter here and call EnvOverride to reuse the implementation.

Reply:
done